Gemfile

```diff
@@ -11,6 +11,7 @@ gem "simple_form"
 gem "slack-ruby-client"
 gem "uglifier"
 gem "sass-rails"
+gem 'omniauth-myusa', git: 'https://github.com/18F/omniauth-myusa.git'
```

Comment:
also, I believe you can do `github` here instead and it will read like

`gem "omniauth-myusa", github: "18F/omniauth-myusa"`